programs/bpf_loader/src/lib.rs

```diff
@@ -147,28 +144,28 @@ fn create_executor_from_bytes(
     register_syscalls_time.stop();
     create_executor_metrics.register_syscalls_us = register_syscalls_time.as_us();
     let mut load_elf_time = Measure::start("load_elf_time");
-    let executable = Executable::<InvokeContext>::from_elf(programdata, loader).map_err(|err| {
+
+    // Allowing mut here, since it may be needed for jit compile, which is under a config flag
+    #[allow(unused_mut)]
+    let mut loaded_program = LoadedProgram::new(
+        loader_key,
+        loader,
+        deployment_slot,
+        programdata,
+        account_size,
+    )
+    .map_err(|err| {
         ic_logger_msg!(log_collector, "{}", err);
         InstructionError::InvalidAccountData
-    });
+    })?;
     load_elf_time.stop();
     create_executor_metrics.load_elf_us = load_elf_time.as_us();
-    let executable = executable?;
-    let mut verify_code_time = Measure::start("verify_code_time");
-    #[allow(unused_mut)]
-    let mut verified_executable =
-        VerifiedExecutable::<RequisiteVerifier, InvokeContext>::from_executable(executable)
-            .map_err(|err| {
-                ic_logger_msg!(log_collector, "{}", err);
-                InstructionError::InvalidAccountData
-            })?;
-    verify_code_time.stop();
-    create_executor_metrics.verify_code_us = verify_code_time.as_us();
+
```

Comment:
This removes the "verify_code_time" metric. Maybe we have to move all three (load, verify and compile) into `LoadedProgram::new()` and have it return `create_executor_metrics`.